va/validation-authority.go

```diff
@@ -564,15 +579,18 @@ type CAASet struct {
 	Unknown   []*dns.CAA
 }
 
-// returns true if any CAA records have unknown tag properties and are flagged critical.
```

Comment:
Please don't add unrelated line wraps like this and the one below. It makes it hard to see the relevant parts of the code. As a general rule: Do drive-by cleanups only on code you already touch. Otherwise, do the cleanup in a separate PR.